js/src/Model/TodoListModel.js

```diff
@@ -0,0 +1,18 @@
+var Backbone = require('backbone');
+// var TodoListCollection = require('../Collection/TodoListCollection.js')
+
+var TodoListModel = Backbone.Model.extend({
+    urlRoot: '/todolists',
+    defaults: {
+        name: 'List Name',
+        creator: 'User'
```

Comment:
As a reminder, you use the same kind of comment for class properties as you would for these default attributes.
```
/**
 * Comment if necessary.
 * @type {data type}
 */
```